test/replication-luatest/gh_7581_downstream_lag_test.lua

```diff
@@ -54,7 +54,10 @@ local function wait_downstream_updated(master, replica)
     local id = replica:get_instance_id()
     t.helpers.retrying({}, function()
         master:exec(function(id)
-            t.assert_equals(box.info.vclock,
+            -- Ignore local lsn
+            local vclock = table.deepcopy(box.info.vclock)
+            vclock[0] = nil
```

Comment:
Why so many changes in the test? This feature shouldn't result in any incompatible user-visible changes.

Reply:
We have lots of tests using vclock. Now local lsn is bumped by WAL GC, so I had to ignore it in many places. Also, in some places I had to manually delete a consumer to delete retained xlogs.

I reduced number of patched test by patching `wait_cluster_vclock` in `test-run` (the last commit).